app/views/declaration/seal_add.scala.html

```diff
@@ -20,36 +20,51 @@
 @import controllers.util.Remove
 @import views.components.fields.LabelledValue
 @import views.Title
+@import controllers.declaration.routes._
+@import controllers.navigation.Navigator
+@import models.requests.JourneyRequest
+@import views.{BackButton, Title}
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import views.components.gds.Styles._
+@import views.html.components.gds._
 
-@this(main_template: views.html.main_template)
+@this(
+        govukLayout: gdsMainTemplate,
+        govukButton: GovukButton,
+        errorSummary: errorSummary,
+        sectionHeader: sectionHeader,
+        exportsInputText: exportsInputText,
+        saveButtons: saveButtons,
+        tariffDetails: tariffDetails,
+        pageTitle: pageTitle,
+        formHelper: uk.gov.hmrc.play.views.html.helpers.FormWithCSRF
+)
 
 @(mode: Mode, form: Form[Seal], containerId: String)(implicit request: Request[_], messages: Messages)
 
 @tariffBody = { @Html(messages("declaration.type.sealsSummaryText", components.details_content_link())) }
 
-@main_template(
-    title = Title(messages("standard.seal.title", containerId))
-) {
-
-    @components.back_link(controllers.declaration.routes.SealController.displaySealSummary(mode, containerId), mode)
+@govukLayout(
+    title = Title("declaration.seal.title", "declaration.seal.summary.heading", containerId),
+    backButton = Some(BackButton(messages("site.back"), controllers.declaration.routes.SealController.displaySealSummary(mode, containerId)))) {
 
-    @components.error_summary(form.errors)
+    @formHelper(action = SealController.submitAddSeal(mode, containerId), 'autoComplete -> "off") {
+        @helper.CSRF.formField
```

Comment:
So we don't need CSRF here

Reply:
As Mauricio pointed, this CSRF formFiled is not necessary

Reply:
`formHelper: uk.gov.hmrc.play.views.html.helpers.FormWithCSRF`